libraries/adaptive-expressions/src/expressionFunctions.ts

```diff
@@ -1962,6 +1962,20 @@ export class ExpressionFunctions {
         return res;
     }
 
+    private static commonStringify(input: any): string {
+        if (input === null || input === undefined) {
```

Comment:
>(input === null || input === undefined [](start = 11, length = 38)

!input would be clearer and more idiomatic. It is also fine for the empty string case.

Reply:
but it is not fine for the boolean type `false`